source/Octopus.Tentacle.Tests.Integration/CapabilitiesServiceV2Test.cs

```diff
@@ -56,21 +56,24 @@ public async Task CapabilitiesResponseShouldBeCached(TentacleConfigurationTestCa
             await using var clientAndTentacle = await tentacleConfigurationTestCase.CreateBuilder()
                 .WithPortForwarder(out var portForwarder)
                 .WithTentacleServiceDecorator(new TentacleServiceDecoratorBuilder()
-                    .HookServiceMethod<IAsyncClientCapabilitiesServiceV2, object, CapabilitiesResponseV2>(nameof(IAsyncClientCapabilitiesServiceV2.GetCapabilitiesAsync),
-                        null,
-                        async (_, response) =>
-                        {
-                            await Task.CompletedTask;
+                    .DecorateCapabilitiesServiceV2With(d => d
+                        .AfterGetCapabilities(
```

Comment:
I think in this case we just want to actually decorate it with the regular decorator pattern e.g.

.DecorateCapabilitiesServiceV2With(d => d.DecorateGetCapabilities(

async (paramasToGetCapabilities, innerGetCapbilitiesService) => {
   // do stuff
    var response = innerGetCapabilitiesService.GetCapabilities(paramasToGetCapabilities);
    capabilitiesResponses.Add(response);
    return response;
    
}

));